x-pack/test_serverless/functional/test_suites/common/platform_security/navigation/avatar_menu.ts

```diff
@@ -13,7 +13,7 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
 
   describe('Avatar menu', function () {
     before(async () => {
-      await svlCommonPage.loginWithRole('viewer');
+      await svlCommonPage.loginWithPrivilegedRole();
```

Comment:
It looks like this function logins in as Editor/Developer based on the project type. This test should log in as Viewer. I think we just need to add a `loginAsViewer` function to the common page utility and use that.

Reply:
That's a good idea.

Reply:
https://github.com/elastic/kibana/pull/188128/commits/badde66df75dea281bfa8a182ee1158b664eb894